governance/votes/Executive vote - July 27, 2022.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Template - [Executive Vote] Real World Assets Vault Onboardings - July 27, 2022
+summary: Onboards Real World Asset Vaults for Huntingdon Valley Bank (HVBank) and Societe Generale – Forge (OFH).
+date: 2022-07-27T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Real World Assets Vault Onboardings - July 27, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Societe Generale – Forge (OFH) will be onboarded as a new vault type, RWA-008, as detailed below. 
+- Huntingdon Valley Bank (HVBank) will be onboarded as a new vault type, RWA-009, as detailed below. 
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### RWA-008 Societe Generale – Forge (OFH) Onboarding
+
+As per [this poll](https://vote.makerdao.com/polling/QmajCtnG#vote-breakdown), RWA-008 Societe Generale – Forge (OFH) will be onboarded with the following parameters, if this executive proposal passes.
+
+#### Collateral Parameters
+
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): 0.05%
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): 30 million DAI
+* [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): 100%
+* Liquidation Process: See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
```

Comment:
SocGen links to the MIPs portal and this one links to the forum, probs best they link to the same source